Rubberduck.Core/UI/Settings/AutoCompleteSettings.xaml

```diff
@@ -112,11 +112,24 @@
                           Content="{Resx ResxName=Rubberduck.Resources.Settings.AutoCompletesPage, Key=ConcatVbNewLine}" />
 
                 <Label Margin="15,0,15,0" Content="{Resx ResxName=Rubberduck.Resources.Settings.AutoCompletesPage, Key=MaxConcatLines}" />
-                <controls:NumberPicker Margin="15,0,15,0" 
-                                       NumValue="{Binding ConcatMaxLines}"
-                                       MinNumber="{Binding ConcatMaxLinesMinValue}"
-                                       MaxNumber="{Binding ConcatMaxLinesMaxValue}"/>
+                <StackPanel Orientation="Horizontal" Margin="15,0,15,0">
+                    <TextBox Margin="5,5,0,5" Height="20" Width="50" Text="{Binding ConcatMaxLines, Mode=TwoWay, StringFormat=\{0:D\}, UpdateSourceTrigger=PropertyChanged, ValidatesOnDataErrors=True}" VerticalAlignment="Top" VerticalContentAlignment="Center" />
+                    <StackPanel Margin="0,5" >
+                        <Button Height="10" Width="20" Command="{Binding IncrementMaxConcatLinesCommand}">
+                            <TextBlock Text="˄" FontSize="10" Margin="0,-4,0,0"/>
+                        </Button>
+                        <Button Height="10" Width="20" Command="{Binding DecrementMaxConcatLinesCommand}">
+                            <TextBlock Text="˅" FontSize="10" Margin="0,-3,0,0"/>
```

Comment:
Could be replaced with ↑↓ or ▲▼ for better visibility (Ascii codes 24,25 / 30,31)